libopenage/log/logsource.cpp

```diff
@@ -21,6 +22,9 @@ void LogSource::log(const message &msg) {
 	// (and thus at least one sink exists).
 	global_stdoutsink();
 
+	// HACK tailsink should probably be initialized somewhere else.
+	global_tailsink();
```

Comment:
The tailsink should be an independent class which is a member of the console. the console then registers itself at the logger to provide the tailsink. You can avoid global state by that way.